app/javascript/pages/Admin/MerchantAccounts/Show.tsx

```diff
@@ -0,0 +1,144 @@
+import { Link, usePage } from "@inertiajs/react";
+import { capitalize } from "lodash";
+import React from "react";
+import { cast } from "ts-safe-cast";
+
+import DateTimeWithRelativeTooltip from "$app/components/Admin/DateTimeWithRelativeTooltip";
+import { BooleanIcon, NoIcon } from "$app/components/Admin/Icons";
+
+export type AdminMerchantAccountProps = {
+  id: number;
+  charge_processor_id: string;
+  charge_processor_merchant_id: string | null;
+  created_at: string;
+  external_id: string;
+  user_id: number | null;
+  country: string;
+  country_name: string | null;
+  currency: string;
+  holder_of_funds: string;
+  stripe_account_url: string | null;
+  charge_processor_alive_at: string | null;
+  charge_processor_verified_at: string | null;
+  charge_processor_deleted_at: string | null;
+  updated_at: string;
+  deleted_at: string | null;
+  live_attributes: { label: string; value: unknown }[];
+};
+
+const AdminMerchantAccountsShow = () => {
+  const { merchant_account } = cast<{ merchant_account: AdminMerchantAccountProps }>(usePage().props);
+
+  return (
+    <div className="override grid gap-4 rounded border border-border bg-background p-4">
+      <div>
+        <h2>Merchant Account {merchant_account.id}</h2>
+        <DateTimeWithRelativeTooltip date={merchant_account.created_at} utc />
```

Comment:
Making sure I didn't miss anything in my other PR(s) - what's the reasoning for using `utc` here (over other places where we don't)?